cmd/lakectl/cmd/root.go

```diff
@@ -131,5 +131,8 @@ func initConfig() {
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_")) // support nested config
 	viper.AutomaticEnv()                                   // read in environment variables that match
 
+	//Configuration defaults
+	viper.SetDefault("server.endpoint_url", "http://127.0.0.1:8000")
```

Comment:
Note that server.endpoint_url is already defined as a constant at the top of this file.
I think the value itself could also go as a separate constant, "DefaultServerEndpointURL".